bin/create-test-application

```diff
@@ -3,18 +3,22 @@
 set -eo pipefail
 
 rn_version=${1:-latest}
-app_name="TestRnApp_$(echo "$rn_version" | tr "." "_")"
+app_name="${INTEGRATION_TEST_APP_NAME:-TestRnApp_$(echo "$rn_version" | tr "." "_")}"
```

Comment:
Being able to set the app's name (and directory) made building the CI config much easier.